devicetypes/smartthings/zwave-panic-button.src/zwave-panic-button.groovy

```diff
@@ -0,0 +1,149 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+
+metadata {
+	definition(name: "Z-Wave Panic Button", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.remotecontroller") {
+		capability "Sensor"
+		capability "Health Check"
+		capability "Battery"
+		capability "Panic Alarm"
+
+		fingerprint mfr: "0258", prod: "0003", model: "108A", deviceJoinName: "NEO SOS"
+		fingerprint mfr: "0258", prod: "0003", model: "108F", deviceJoinName: "NEO SOS"
+	}
+
+	tiles(scale: 2) {
+		multiAttributeTile(name: "panic", type: "generic", width: 6, height: 4) {
+			tileAttribute("device.panicAlarm", key: "PRIMARY_CONTROL") {
+				attributeState "panic", label: "Emergency!", icon:"st.alarm.beep.beep", backgroundColor:"#bc2f2f"
+				attributeState "clear", label: "", icon:"st.Weather.weather13", backgroundColor:"#a4a9aa"
+			}
+		}
+
+		childDeviceTiles("buttons")
+
+		valueTile("battery", "device.battery", decoration: "flat", inactiveLabel: false, width: 2, height: 2) {
+			state "battery", label: '${currentValue}% battery', unit: ""
+		}
+
+		main(["panic"])
+
+		details(["panic", "buttons", "battery"])
+	}
+}
+
+def installed() {
+	log.debug "Installed $device.displayName"
+	sendEvent(name: "DeviceWatch-Enroll", value: [protocol: "zwave", scheme:"untracked"].encodeAsJson(), displayed: false)
```

Comment:
this is deprecated

You'll want to set a checkInterval of (presumably) 8 hours and 2 minutes.

Reply:
Thing is, I don't know what wakeUpInterval this device has. When asked with *Wake Up Interval Report* it reports 0 seconds.

Reply:
In 75983b0 commit I've just commented this line, as I'm not sure how to handle this "not-so-waking-up" case. 
I mean this device is waking up, but not in constant interval. I think it is happening when there wasn't any interaction with device for a long time, button is pushed and then device is sending wakeUpNotification. Although I'm not 100% sure about that, I'll need to investigate it further.

Reply:
We automatically set a wakeup interval of 4 hours to all devices that claim they support it during join. If it's not durable that makes me nervous. Watch hub logs during join and you should see the device's minimum and maximum wakeup interval values. It might be that it doesn't allow a value <= 4 hours.